Tensile/TensileCreateLibrary.py

```diff
@@ -141,7 +141,7 @@ def getAssemblyCodeObjectFiles(kernels, kernelWriterAssembly, outputPath):
         else:
           newCOFiles = [os.path.join(destDir, k + '_' + archName + '.co') for k in assemblyKernelNames]
 
-        for src, dst in Utils.tqdm(zip(origCOFiles, newCOFiles), "Copying code objects"):
+        for src, dst in zip(origCOFiles, newCOFiles) if globalParameters["PrintLevel"] == 0 else Utils.tqdm(zip(origCOFiles, newCOFiles), "Copying code objects"):
```

Comment:
Is this the replacement for the copy targets?

Reply:
Utils.tqdm outputs a progress bar so we are asking if we want to silence all output and if so don't use the progress bar.